adafruit_ble/__init__.py

```diff
@@ -162,14 +162,16 @@ def __init__(self, adapter=None):
         self._current_advertisement = None
         self._connection_cache = {}
 
-    def start_advertising(self, advertisement, scan_response=None, interval=0.1):
+    def start_advertising(self, advertisement, scan_response=None, interval=0.1, timeout=0):
```

Comment:
Use `timeout=None` to indicate no timeout. That is more Pythonic, and the way we do it in other similar API's.